go/rds/README.md

```diff
@@ -0,0 +1,123 @@
+# AWS SDK for Go Code Examples for Amazon RDS
```

Comment:
sentence case the title "...**code examples**

Reply:
Fixed